panel/_templates/autoload_panel_js.js

```diff
@@ -27,6 +27,8 @@ calls it with the rendered model.
   const py_version = '{{ version }}'.replace('rc', '-rc.').replace('.dev', '-dev.');
   const reloading = {{ reloading|default(False)|json }};
   const Bokeh = root.Bokeh;
+  const BK_RE = /^https:\/\/cdn\.bokeh\.org\/bokeh\/release\/bokeh-/;
+  const PN_RE = /^https:\/\/cdn\.holoviz\.org\/panel\/[^/]+\/dist\/panel/i
```

Comment:
How would this work with inline and a local server? Mostly interested in the former tbh.